src/Footer.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
```

Comment:
This looks good on your page! 

Two suggestions for improvements:
- If you open devtools on the browser you will see that there is an error being thrown in the console that looks like
```
index.js:1 Warning: Each child in a list should have a unique "key" prop.
```
It is because you need a `key` for each element in your array. [See these notes from class](https://syllabus.codeyourfuture.io/react/week-1/lesson#keys).
-Instead of using a `<div>` it would be better semantically to use the `<footer>` tag. It won't change any visually on the screen, but improve the accessibility of your page (screen readers for people who are blind rely on semantics in pages to describe the page better for them).

Reply:
Note: Your code is still throwing this error in `Footer.js`
```
Warning: Each child in a list should have a unique "key" prop.
```